app/inc/constants.php

```diff
@@ -28,7 +28,8 @@
 }
 
 // Special modes for the app
-define('DEBUG', (strstr(VERSION, 'dev') || isset($_GET['debug'])) ? true : false);
+define('DEBUG',     strstr(VERSION, 'dev') || isset($_GET['debug']));
+define('LOCAL_DEV', isset($server_config['dev']) && $server_config['dev']);
```

Comment:
Wow, never noticed it. Are we already using it somewhere? Can't find anything in the code.

Can you add `dev=false`to config.ini-dist? I also wonder if we should just use one system to determine if it's dev or not (I always considered the version to be the trigger, but I like a specific parameter a lot more).